packages/dds/tree/src/test/edit-manager/editManager.spec.ts

```diff
@@ -322,10 +323,11 @@ describe("EditManager", () => {
 			assert.deepEqual(anchors.intentions, [1, 3]);
 
 			const inverseChanges = changes.map(TestChange.invert).reverse();
-			const inverseDeltas = inverseChanges.map(family.intoDelta);
-			const deltas = Array.from(manager.rollbackLocalChanges(startingRevision));
-			assert.deepEqual(deltas, inverseDeltas);
-			const totalRebases = [...changes, ...inverseChanges];
+			const composedInverse = TestChange.compose(inverseChanges.map(makeAnonChange));
+			const inverseDelta = family.intoDelta(composedInverse);
+			const delta = manager.rollbackLocalChanges(startingRevision);
+			assert.deepEqual(delta, inverseDelta);
```

Comment:
deltas can contain cursors right? Thus I don't think deepEqual is a good way to compare them.

Maybe we should have a deep semantic compare for them, as well as a function to assert equality that prints differences nicely like we have for UpPaths? At least for tests having a way to clone them into some normalizes json compatible objects then using assert.deepEqual can be clean (thats what we did for UpPaths)

Reply:
We do have a helper for that. It doesn't acutally matter here because the delta will only contain plain values (courtesy of `TestChange`), but it's good to establish the right pattern anyway.